### ISSUE TITLE
kraken and kraken2 recipes overwrite each other's files (impacts bracken recipe)

### ISSUE THREAD

**Issue by pvanheus** [bioconda/bioconda-recipes#17791]
The (kraken)[https://github.com/bioconda/bioconda-recipes/blob/master/recipes/kraken/build.sh] and (kraken2)[https://github.com/bioconda/bioconda-recipes/blob/master/recipes/kraken2/build.sh] build recipes both install to `$PREFIX/libexec`. Some of the files they install have the same names, e.g. `$PREFIX/libexec/standard_installation.sh`. The result is that if they are installed together (as happens when, e.g. installing (bracken)[https://github.com/bioconda/bioconda-recipes/blob/master/recipes/bracken/meta.yaml]) the two installations overwrite each other with unpredictable results.

Proposed solutions: install files to `$PREFIX/libexec/kraken2` and `$PREFIX/libexec/kraken`. @dfornika you might be interested in this. :)

**Comment by yesimon** [comment]
See https://github.com/DerrickWood/kraken2/pull/91. I don't think placing them in a subdirectory of `libexec` will work.

**Comment by pvanheus** [comment]
Yes, you are right - the `kraken2-build` etc perl scripts refer to the files by "base" name throughout.

**Comment by dfornika** [comment]
Thanks for tagging me @pvanheus.

Maybe the simplest solution would be to drop `kraken` as a dependency and just leave `kraken2`?

**Comment by dfornika** [comment]
@pvanheus @yesimon I've submitted a PR to remove `kraken` as a dependency (#17921). Please review if you agree with that solution.
